types/types.go

```diff
@@ -509,6 +509,10 @@ type SystemContext struct {
 	SystemRegistriesConfPath string
 	// Path to the system-wide registries configuration directory
 	SystemRegistriesConfDirPath string
+	// Path to the user-specific aliases configuration file
+	UserAliasesConfPath string
```

Comment:
(Also mentioned elsewhere:) This probably should talk about short names, to disambiguate. And …somehow… warn that aliases are only applicable to a specific subpackage, not to most of c/image.